src/api/app/assets/stylesheets/webui/responsive_ux/grid.scss

```diff
@@ -27,31 +27,22 @@
 
 @include media-breakpoint-up(xl) {
   #grid {
-    grid-template-columns: $left-navigation-width auto;
-    grid-template-rows: $top-navigation-height auto;
+    grid-template-columns: min-content auto;
+    grid-template-rows: $top-navigation-height 1fr auto;
     grid-template-areas:
       'top-navigation top-navigation'
-      'left-navigation content';
+      'left-navigation content'
+      'toggle-sidebar-button content';
 
     #left-navigation-area {
       grid-area: left-navigation;
       display: block;
+      grid-template-columns: min-content auto;
+    }
 
-      #left-navigation {
-        position: sticky;
-        top: $top-navigation-height;
-        left: 0;
-        max-height: calc(100vh - #{$top-navigation-height});
-        overflow-y: auto;
-        overflow-x: hidden;
-
-        & a {
```

Comment:
I reverted back the styles for word-wrap on links. I also kept 'width' here, because if remove it, the position of characters will be recalculated each time, when sidebar is expanded.
![Peek 2020-10-14 14-28](https://user-images.githubusercontent.com/37581072/95988738-a1724280-0e29-11eb-976e-ecfba8bbbc74.gif)